frontend/components/TargetLabelSelector/TargetLabelSelector.tests.tsx

```diff
@@ -5,31 +5,61 @@ import { noop } from "lodash";
 import TargetLabelSelector from "./TargetLabelSelector";
 
 describe("TargetLabelSelector component", () => {
-  it("renders the custom target selector when the target type is 'Custom'", () => {
-    render(
-      <TargetLabelSelector
-        selectedTargetType="Custom"
-        selectedCustomTarget="labelIncludeAny"
-        customTargetOptions={[
-          { value: "labelIncludeAny", label: "Include any" },
-        ]}
-        selectedLabels={{}}
-        labels={[
-          { id: 1, name: "label 1", label_type: "regular" },
-          { id: 2, name: "label 2", label_type: "regular" },
-        ]}
-        onSelectCustomTarget={noop}
-        onSelectLabel={noop}
-        onSelectTargetType={noop}
-      />
-    );
+  describe("renders the custom target selector when the target type is 'Custom'", () => {
```

Comment:
TIL you can nest `describe`s!

Reply:
Yes! And depending on the test reporterit will concatenate them in the output, e.g.

"TargetLabelSelector component renders the custom target selector when the target type is 'Custom' with a dropdown when there are custom options to choose from"

And you can use `before/after/each/all` inside the nested block.

Reply:
@fleetdm/frontend ^